web/app/js/components/util/ApiHelpers.jsx

```diff
@@ -97,6 +97,7 @@ const ApiHelpers = (pathPrefix, defaultMetricsWindow = '1m') => {
   };
 
   const urlsForResource = (type, namespace) => {
+    // Traffic Performance Summary. This retrieves stats for the given resource.
```

Comment:
Should we also document that if type == 'all' then we get stats for all resources?